vivisect/tests/testvivisect.py

```diff
@@ -133,8 +133,8 @@ def test_vaset_populate(self):
                               (134549928,), (134544175,), (134553138,), (134572852,), (134580661,),
                               (134550582,), (134579529,), (134572857,), (134534204,), (134577471,), (134580544,),
                               (134568769,), (134546505,), (134578743,), (134553208,), (134555349,),
-                              (134580057,), (134521306,), (134531547,), (134572084,), (134578576,), (134554727,),
```

Comment:
any idea what these changes are?

Reply:
So this is the reason why this is still a WIP. The fix I have in noret precludes marking `vdir.print_and_abort` as no return since `print_and_abort` ends up in the entrypoints list. (those va that I removed arethe obstack_alloc_failed_handler with emulation ends up pointing to print and abort).

I had a thought that the fix could also be in the emulator. Because the ultimate problem is that certain functions in the entrypoint were getting marked as noreturn (which looking at the functions was understandable), but then the emulator would refuse to run them because the first va was marked as no return and would bail out early, cutting emulation analysis short (which things like libc relied on). and check to make sure we can get past the first instruction in the emulator (or even a configurable override) would be nice.

The old elf plt module helped skip past this. But I was toying around with the above last night and it seems to work. I'm double checking it today and then I might back this particular test change out. I actually forgot I had this one in here while I was trying to get past this failure to see if the other va sets were kosher.

Reply:
weee!  and i no longer feel like a jerk for pointing this out. :)

Reply:
Ah, it's usually good to bring these up. Forces me to double check the work I did late at night.